stdlib/LinearAlgebra/test/qr.jl

```diff
@@ -71,6 +71,11 @@ rectangularQ(Q::LinearAlgebra.AbstractQ) = convert(Array, Q)
                 rstring  = sprint((t, s) -> show(t, "text/plain", s), r)
                 qstring  = sprint((t, s) -> show(t, "text/plain", s), q)
                 @test qrstring == "$(summary(qra))\nQ factor:\n$qstring\nR factor:\n$rstring"
+                # iterate
+                q, r = qra
+                @test q*r ≈ a
+                # property names
+                @test Base.propertynames(qra)       == (:R, :Q)
```

Comment:
Not that it matters, but what is this extensive space good for? 😛

Reply:
I must have hit tab...